### PR TITLE
Make our links more accessible.

### PR DESCRIPTION
**Pull request by sadleb**
### Description
Task: https://app.asana.com/0/1174274412967132/1200411617157219

#### Confirmation failed - BEFORE
image

#### Confirmation failed - AFTER
image

#### Hustle To Career Project - AFTER
image

#### Link in a Module for reference
image

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added neccessary labels
- [ ] My PR title follows the convention in the guide
- [ ] I have assigned myself to the PR
- [ ] I have requested reviews from team members
- [ ] I have created a task for reviewers on PM tool
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/assets/stylesheets/shared/style.scss [hunk 1]
```diff
@@ -74,12 +74,9 @@ p {
...
 
-a {
-	color: $bv-oceanblue-600;
-}
-
+a,
 a:visited {
-	color: $bv-oceanblue-700;
+  color: $bv-oceanblue-500;
 }
```
Comment (sadleb) [inline app/assets/stylesheets/shared/style.scss:80]: I didn't think it was worth trying to find two accessible colors for visited vs not. I think that was just carried over from some legacy Join server styling. I don't think it's actually needed or adds value. Do you disagree?

Comment (rshipp) [reply app/assets/stylesheets/shared/style.scss]: https://webaim.org/blog/wcag-2-0-and-link-colors/ doesn't list :visited style as a requirement, and I don't see other things saying it is in a quick search. Do we just not like the default purple? Color-only indicators aren't accessible anyway so I think it's more a UX issue than an accessibility one.

**Review by rshipp — APPROVED** [review]

**Comment by rshipp** [comment]
i noticed in one of your screenshots we say "opens in new tab" for an email link, which doesn't make sense. followup for that here https://app.asana.com/0/1174274412967132/1200701153958445/f

File: app/assets/stylesheets/shared/style.scss (hunk 1 above)
Comment (sadleb) [reply app/assets/stylesheets/shared/style.scss]: Whoops, that comment is outdated. I originally did a color only fix and backed that out in favor of underlines. I’m just not touching the old colors at this point.

**Pull request merged**